src/http-api.ts

```diff
@@ -5,25 +5,35 @@ import responseTime from 'response-time';
 
 import cors from 'cors';
 import express from 'express';
-import { register, Histogram } from 'prom-client';
+import { register, Histogram, Counter } from 'prom-client';
+import morgan from 'morgan';
 
 import { v4 as uuidv4 } from 'uuid';
 
 import { jsonRpcApi } from './json-rpc-api.js';
+import { JsonRpcRequest } from './lib/json-rpc.js';
 import { createJsonRpcErrorResponse, JsonRpcErrorCodes } from './lib/json-rpc.js';
 
 export class HttpApi {
+  requestCounter = new Counter({
+    name       : 'dwn_requests_total',
+    help       : 'all dwn requests processed',
+    labelNames : ['method', 'status', 'error'],
+  });
+
   api: Express;
   dwn: Dwn;
 
   constructor(dwn: Dwn) {
     this.api = express();
     this.dwn = dwn;
 
+    this.api.use(morgan('tiny')); // custom log formats are fairly easy, if we want to use a different format: https://expressjs.com/en/resources/middleware/morgan.html
```

Comment:
i think this will work for now. though, if we wanted to go beyond just request logging and add debug logs into individual request handlers, i'd reach for something like [loglevel](https://www.npmjs.com/package/loglevel) and various loglevel plugins like [loglevel-plugin-prefix](https://www.npmjs.com/package/loglevel-plugin-prefix) to set up a logger like so:

```typescript
import log from 'loglevel'
import prefix from 'loglevel-plugin-prefix'

import { config } from './config.js'

log.setLevel(config.logLevel)

prefix.reg(log)
prefix.apply(log)

export default log
```

this can then be used to log requests and as needed in request handlers and other parts of code

Reply:
I have updated the PR to use `loglevel`, which also lets us log response time as part of the existing response time measurement. Let me know what you think.